packages/core/src/lib/export-import/import/import.service.ts

```
@@ -1131,19 +190,19 @@
-	async migrateImportEntityRecord(item: IRepositoryModel<any>, entity: any): Promise<any> {
-		const { repository, uniqueIdentifier = [] } = item;
+	async migrateImportEntityRecord(item: IRepositoryModel, entity: any): Promise<any> {
+		const { repository, uniqueIdentifiers = [] } = item;
 		const masterTable = repository.metadata.tableName;
 
 		return await new Promise(async (resolve, reject) => {
 			try {
 				const source = JSON.parse(JSON.stringify(entity));
 				const where = [];
-				if (isNotEmpty(uniqueIdentifier) && uniqueIdentifier instanceof Array) {
+				if (isNotEmpty(uniqueIdentifiers) && uniqueIdentifiers instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)